### DISCUSSION TITLE
how do i change the alpha value to 251 not 255

### DISCUSSION THREAD

**Discussion by SNokerYT** [ImageMagick/ImageMagick#5684] [Help]
how do I change the change alpha value to 251 | like this image 
i also have some images doesn't have a alpha value
![image](https://user-images.githubusercontent.com/68678154/196944573-3a73f02d-e7a2-4e6d-8e19-eccacd7b143f.png)
![image](https://user-images.githubusercontent.com/68678154/196944400-57985a26-ae1f-413c-845a-0486104b57e5.png)
![image](https://user-images.githubusercontent.com/68678154/196944214-2c134f5e-4467-4633-8891-12faf441d5fc.png)

**Comment by snibgo** [comment]
What version of IM, on what platform?

If your image doesn't have an alpha channel, "`-alpha set`" will add one. See https://imagemagick.org/script/command-line-options.php#alpha

IM record values in the nominal range of 0 to QuantumRange, where QuantumRange may be 255 or 65535 or something larger.

In an "`-fx`" expression, values are normalised to the nominal range 0.0 to 1.0.

```
magick in.png -alpha set -channel A -fx "u==1?251/255:u" +channel out.png
```
This limits the "`-fx`" operation to just the "A" channel, Alpha. Where the value is 1, it is change to 251/255. Otherwise it is unchanged.

(If your IM is old, the -fx will be slow.)
